### PR TITLE
fix: id:55035 banner icon color correction

### PR DESCRIPTION
**Pull request by Farhan-emgage**
banner icon color correction

### CODE REVIEW

File: src/components/Banner/Banner.scss [hunk 1]
```diff
@@ -51,18 +52,22 @@ $secondary-action-horizontal-padding: 1.5 * spacing(tightest);
 
 .statusSuccess {
   @include banner-colors(color(green), color(green, lighter));
+  @include icon-color(color(banner, success));
```
Comment (weegrom) [inline src/components/Banner/Banner.scss:55]: Why not follow the same pattern as is used for the other colors in the banner? Eg: @include icon-color(color(green, dark));

**Comment by emgageadmin** [comment]
https://atera.tpondemand.com/entity/55035

**Comment by weegrom** [comment]
@Farhan-emgage please set up a meeting to discuss color usage. I feel you need additional training of color palette purpose/usage. Thanks

**Review by weegrom — APPROVED** [review]
@Farhan-emgage good job with this one. I see you implement as we spoke about. You just need to do the same for the SwitchCheckbox component. Thanks

**Pull request merged**